html/includes/modal/alert_template.inc.php

```diff
@@ -37,6 +37,10 @@
                             <label for="template">Template: </label>
                             <textarea class="form-control" id="template" name="template" rows="15"></textarea>
                         </div>
+                        <div class="form-group">
+                            <label for="rules_list">Attach template to rules: </label>
+                            <select id="rules_list" name="rules_list" class="form-control" multiple data-live-search="true" data-max-options="10"></select>
```

Comment:
Do we need to limit to 10 here?

Reply:
I have kept the [old behavior](https://github.com/librenms/librenms/blob/master/html/includes/modal/attach_alert_template.inc.php#L35) but we can definitely remove this limit.